pkgs/development/tools/ansible-lint/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, pythonPackages, fetchurl, ansible }:
+
+pythonPackages.buildPythonPackage rec {
+  name = "ansible-lint-${version}";
+  version = "3.4.20";
+
+  src = fetchurl {
+    url = "https://github.com/willthames/ansible-lint/archive/v3.4.20.tar.gz";
+    sha256 = "b7fb92f00b9723e3a9624d70b53b60db41426c0ef4fad05ab28c7e4f28a0b6e2";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ pyyaml six ] ++ [ ansible ];
```

Comment:
This is going to be interesting, as `ansible` is used here as a module. In order to make this reliably work we should make `ansible` a Python library instead of an application.